lib/releasy.js

```diff
@@ -33,6 +33,10 @@ module.exports = function (opts) {
 
   // Pachamama v2 requires that version tags start with a 'v' character.
   config.tagName = `v${config.newVersion}`
+  if (opts.displayName) {
+    config.tagName = `${config.tagName} of ${versionProvider.readName()}`
```

Comment:
I think `packageName@version` is a more widespread "syntax" of release tags for monorepos, and it is more easily parsed by custom tools

```suggestion
    config.tagName = `${versionProvider.readName()}@${config.tagName}`
```

Reply:
I'm just worried with this comment: "Pachamama v2 requires that version tags start with a 'v' character." The release it would like: "vvtex.subscriptions@2.2.3"

Reply:
so it looks like we should update pachamama to handle this case of a monorepo?

Reply:
That's out of my league, monorepo is still experimental. What about this:
```suggestion
    config.tagName = `v ${versionProvider.readName()}@${config.tagName}`
```

It would look like this: v vtex.subscriptions@2.2.3

Reply:
Since this is an optional feature, I've changed to your suggested format, I've talked to @pipinho, but this will be only implemented in the future.... I've added a TODO comment to remind our future selves....